adapters/Tapjoy/BuildScript/README.md

```diff
@@ -0,0 +1,25 @@
+# Steps to generate universal static library and framework for iOS Devices and
+Simulators with supported architectures.
```

Comment:
@amadkhilji Can you please clarify? I'm not clear what update needs to be done here. With the two commits that you pushed, the first one `f93b7d0` looked good to me, but it was changed again in your second commit `87298fc` causing the second line to not be formatted correctly.